lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -453,19 +454,35 @@ spec = do
         (_, p:_) <- eventually $
             unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
         w <- emptyWallet ctx
-        r <- joinStakePool ctx (p ^. #id) (w, "Secure Passprase")
-        expectResponseCode HTTP.status403 r
+        joinStakePool ctx (p ^. #id) (w, "Secure Passprase") >>= flip verify
+            [ expectResponseCode HTTP.status200
+            , expectFieldEqual status Pending
+            , expectFieldEqual direction Outgoing
+            ]
 
-        rq <- quitStakePool ctx (p ^. #id) (w, "Secure Passprase")
-        expectResponseCode HTTP.status501 rq
+        -- Wait for the certificate to be inserted
+        eventually $ do
+            let ep = listTxEp w mempty
+            request @[ApiTransaction n] ctx ep Default Empty >>= flip verify
+                [ expectListItemFieldEqual 0 direction Outgoing
+                , expectListItemFieldEqual 0 status InLedger
+                ]
+
+        request @ApiWallet ctx (getWalletEp w) Default Empty >>= flip verify
+            [ expectFieldEqual delegation (Delegating (p ^. #id))
+            ]
+
+        r <- quitStakePool ctx (p ^. #id) (w, "Secure Passprase")
+        expectResponseCode HTTP.status200 r
```

Comment:
At this moment I get here 403 :thinking: investigating ..

Reply:
Please, look at my last comment :)